.all-contributorsrc

```diff
@@ -176,6 +176,15 @@
       "contributions": [
         "code"
       ]
+    },
+     {
```

Comment:
Please remove this part - the bot does it automatically and this will break everything :)